kolena/annotation.py

```diff
@@ -55,6 +57,7 @@ class _AnnotationType(DataType):
     BITMAP_MASK = "BITMAP_MASK"
     LABEL = "LABEL"
     TIME_SEGMENT = "TIME_SEGMENT"
+    TEXT_SEGMENTS = "TEXT_SEGMENTS"
```

Comment:
Should be singular (`TEXT_SEGMENT`) now that the annotation is simplified.

Reply:
got it